Source/Details/_ASDisplayView.h

```diff
@@ -39,6 +39,14 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)__forwardTouchesEnded:(NSSet *)touches withEvent:(UIEvent *)event;
 - (void)__forwardTouchesCancelled:(NSSet *)touches withEvent:(UIEvent *)event;
 
+// These methods expose a way for ASDisplayNode responder methods to let the view call super responder methods
```

Comment:
This first sentence isn't quite clear -- "for ASDisplayNode responder methods to let the view call super responder methods".

Clarify if possible, otherwise not exactly sure how to improve it.

Reply:
The sentence is kind of the same as the one for all of the forwarding touch events. In that case we should find a better way for both.